src/management/index.js

```diff
@@ -299,26 +299,15 @@ var ManagementClient = function(options) {
  * @return {Object}   Object containing client information.
  */
 ManagementClient.prototype.getClientInfo = function() {
-  var clientInfo = {
+  return {
```

Comment:
Seems like it would be a good idea to pull this and `AuthenticationClient.prototype.getClientInfo` from the same place...